docs/haproxy-conf.md

```diff
@@ -83,14 +83,23 @@ Edit the `deploy/cr.yaml` Custom Resource manifest and specify the following con
 ```yaml
 spec:
   haproxy:
-    exposePrimary:
-      enabled: true
-      type: LoadBalancer
-      loadBalancerSourceRanges:
+    enabled: true
```

Comment:
@nastena1606 please check these lines https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/cr.yaml#L284-L294 we depreciate only one option "enabled: true"